winbuild/winbuild.go

```diff
@@ -0,0 +1,59 @@
+package winbuild
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+)
+
+const (
+	buildNumAPIUrl = "https://keybase.io/_/api/1.0/pkg/build_number.json"
+)
+
+type buildNoResponse struct {

```

Comment:
Does `buildNo` mean `buildNumber`? If so, would prefer `buildNumberResponse` or `buildNumResponse`, since `buildNoResponse` sounds like "build" + "no response".